R-package/README.md

```diff
@@ -0,0 +1,564 @@
+
+[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/RGF)](http://cran.r-project.org/package=RGF)
+[![Travis-CI Build Status](https://travis-ci.org/mlampros/RGF.svg?branch=master)](https://travis-ci.org/mlampros/RGF)
+[![codecov.io](https://codecov.io/github/mlampros/RGF/coverage.svg?branch=master)](https://codecov.io/github/mlampros/RGF?branch=master)
+[![Downloads](http://cranlogs.r-pkg.org/badges/grand-total/RGF?color=blue)](http://www.r-pkg.org/pkg/RGF)
+
+
+## RGF (Regularized Greedy Forest)
+<br>
+
+
+The **RGF** package is a wrapper of the [Regularized Greedy Forest (RGF)](https://github.com/RGF-team/rgf_python) *python* package, which also includes a [Multi-core implementation (FastRGF)](https://github.com/baidu/fast_rgf). More details on the functionality of the RGF package can be found in the [blog-post](http://mlampros.github.io/2018/02/14/the_RGF_package/) and in the package Documentation. 
+
+<br>
+
+**UPDATE 26-07-2018**: A [Singularity image file](http://mlampros.github.io/2018/07/26/singularity_containers/) is available in case that someone intends to run *RGF* on Ubuntu Linux (locally or in a cloud instance) with all package requirements pre-installed. This allows the user to utilize the *RGF* package without having to spend time on the installation process.
+
+<br>
+
+**References:**
+
+[Rie Johnson and Tong Zhang, Learning Nonlinear Functions Using Regularized Greedy Forest](https://arxiv.org/abs/1109.0887)
+
+https://github.com/RGF-team/rgf/tree/master/RGF
+
+https://github.com/RGF-team/rgf/tree/master/FastRGF
+
+https://github.com/RGF-team/rgf/tree/master/python-package
+
+
+<br>
+
+### **System Requirements**
+
+<br>
+
+* Python (2.7 or >= 3.4)
+
+
+<br>
+
+All modules should be installed in the default python configuration (the configuration that the R-session displays as default), otherwise errors will occur during the *RGF* package installation (**reticulate::py_discover_config()** might be useful here). 
+
+<br>
+
+#### **Debian/Ubuntu/Fedora**    [ Python 2.7 ]
+
+<br>
+
+First install / upgrade the dependencies,
+
+<br>
+
+```R
+sudo pip install --upgrade pip setuptools
+
+sudo pip install -U numpy
+
+sudo pip install --upgrade scipy
+
+sudo pip install -U scikit-learn
+```
+
+<br>
+
+Then, download the *rgf-python* package and install it using the following commands,
+
+<br>
+
+
+```R
+
+git clone https://github.com/RGF-team/rgf.git
+
+cd rgf/python-package
+
+sudo python setup.py install
+
+```
+
+<br>
+
+*FastRGF* will be installed successfully only if gcc >= 5.0.
+
+<br><br>
+
+
+
+#### **Macintosh OSX**            [ Python >= 3.4 ]
```

Comment:
Can you please elaborate why RGF can't work on macOS with Python 2.7?

Reply:
I already mentioned that the RGF R package can be installed on 

```R
Python (2.7 or >= 3.4)

```
The version on the brackets mentions the Python version that I personally installed / tested the RGF R package.

Reply:
It's not obvious. You should explicitly write about it. Because it's can be interpreted that on Linux RGF works with Python (2.7 or >= 3.4), but on MacOS only with  Python (>= 3.4).

Reply:
yes you are right on this.